src/store.js

```diff
@@ -1,6 +1,7 @@
 import { hashHistory } from 'react-router'
 import { createStore, applyMiddleware, compose } from 'redux'
 import thunk from 'redux-thunk'
+import logger from 'redux-logger'
```

Comment:
Why do you use `redux-logger`? The redux dev tools are much more in depth: https://github.com/gnosis/gnosis-management/pull/65/files#diff-3bf35cd41da51153dcca3003656f7c67R35